WooCommerce/UITestsFoundation/Screens/Orders/SingleOrderScreen.swift

```diff
@@ -6,15 +6,28 @@ public final class SingleOrderScreen: ScreenObject {
     // TODO: Remove force `try` once `ScreenObject` migration is completed
     let tabBar = try! TabNavComponent()
 
+    private let editOrderButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["order-details-edit-button"]
+    }
+
+    private let summaryCellGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["summary-table-view-cell-title-label"]
+    }
+
     private let collectPaymentButtonGetter: (XCUIApplication) -> XCUIElement = {
         $0.buttons["order-details-collect-payment-button"]
     }
 
+    private var editOrderButton: XCUIElement { editOrderButtonGetter(app) }
+
     private var collectPaymentButton: XCUIElement { collectPaymentButtonGetter(app) }
 
     public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
-            expectedElementGetters: [ { $0.staticTexts["summary-table-view-cell-title-label"]} ],
+            expectedElementGetters: [
+                summaryCellGetter,
+                editOrderButtonGetter
```

Comment:
nit: I'd advocate for using a single expected element in a screen object's init method, when possible, especially if a specific test will exercise the other elements we expect to have on the screen.

Having more expected elements here slows down the test runs, because each element has to be checked before moving on with the test steps. Ideally we would choose a unique element to determine whether or not the expected screen is loaded. Other elements will be checked as we interact with them in a particular test.

Reply:
As far as I know - only the first element is checked during `init`. When we want to validate all expected elements, we have to make dedicated call, that's why we had verifySingleOrderScreenLoaded() added to test_load_orders_screen.

https://github.com/Automattic/ScreenObject/blob/e27405a65672c62e5a055697eafdeaf073ea63ff/Sources/ScreenObject/ScreenObject.swift#L76

But since we've added edit flow navigation in different test, we can remove both `verifySingleOrderScreenLoaded()` and additional expected element to reduce test time and remove duplicated check.

Reply:
Ah! I see that check changed as of https://github.com/Automattic/ScreenObject/pull/11; I missed that change. That's great!